src/interface/mod.rs

```diff
@@ -16,4 +16,5 @@ pub use pipe::*;
 pub use misc::*;
 pub use timer::*;
 pub use types::*;
-pub use errnos::*;
\ No newline at end of file
+pub use errnos::*;
+pub use std::sync::atomic::{AtomicI32, Ordering};
```

Comment:
So we don't actually declare things in this mod file, its there to combine the interface files into an easy declaration. We also change the names from the std definition.

@jesings I think you're also importing atomic types in networking. Do you have any suggestion to how we could declare these all in one place?

Reply:
They're all being imported in misc.rs within mod, if you just add a new element to that line (importing AtomicI32 as RustAtomicI32, I also deal with ordering in networking so you can look there for examples).

Reply:
Okay, removed from mod.rs; moved to misc.rs.